### PR TITLE
Tweaking in several places

### PR DESCRIPTION
**Pull request by thormeier**
Tweaking of git persistence. Saw some stuff while actually using it that could be fixed.

### CODE REVIEW

File: Command/GitPersistenceInitCommand.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $persistence = $this->getPersistence();
 
-        if (null !== $remote) {
+        if (null === $remote) {
```
Comment (lsmith77) [inline Command/GitPersistenceInitCommand.php]: this looks like a bug fix .. right?

File: Command/GitPersistenceAwareCommand.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
  *
  * @license http://opensource.org/licenses/MIT MIT License
  * @author Pascal Thormeier <pascal.thormeier@liip.ch>
- * @copyright Copyright (c) 2013, Liip, http://www.liip.ch
+ * @copyright Copyright (c) 2014, Liip, http://www.liip.ch
```
Comment (dbu) [inline Command/GitPersistenceAwareCommand.php]: this should probably be done over all files, and put to 2015 while we are at it

Comment (lsmith77) [inline Command/GitPersistenceAwareCommand.php]: well the file was added in 2014 .. in general it should then become a range .. ie. 2014-2015 .. but imho its all a bit BS and I never had any case where it mattered ..

**Comment by lsmith77** [comment]
needs a rebase

**Comment by thormeier** [comment]
Rebased

**Pull request merged**